environment.yaml

```
@@ -0,0 +1,8 @@
+channels:
+  - defaults
+  - conda-forge
+dependencies:
+  - pip
+  - python=3.8
+  # Only put dependencies that are NOT available with pip here (e.g. graph-tool)
```

For M1 compatibility

```suggestion
  - numpy
  - scipy
  # Only put dependencies that are NOT available with pip here (e.g. graph-tool)
```